src/gp_rpc_import_and_canon_name.c

```diff
@@ -17,6 +17,8 @@ int gp_import_and_canon_name(struct gp_call_ctx *gpcall UNUSED,
     gss_OID mech = GSS_C_NO_OID;
     gss_name_t import_name = GSS_C_NO_NAME;
     gss_name_t output_name = GSS_C_NO_NAME;
+    gss_buffer_desc localname = { 0 };
```

Comment:
Technically this should be `GSS_C_EMPTY_BUFFER`; in practice it's going to be the same thing.

Reply:
I'll fix for consistency anyway